vscode/src/client/analyzerClient.ts

```diff
@@ -441,8 +441,8 @@ export class AnalyzerClient {
           const requestParams = {
             label_selector: getConfigLabelSelector(),
             included_paths: filePaths?.map((uri) => uri.fsPath),
+            reset: !(filePaths && filePaths.length > 0),
```

Comment:
@shawn-hurley 
we have a command to clear analysis results in IDE. We should either clear the cache from that command or clear it on the next partial analysis. Otherwise discarded results will get resurrected after file save :)

Reply:
Yep. I think we should be able to clear it on demand.
However this is not a new problem - I've tested it on older version (before this PR).
We can fix it in follow-up  https://github.com/konveyor/editor-extensions/issues/356